src/hooks/protections/net_bundle_get_data_size.cpp

```diff
@@ -0,0 +1,20 @@
+#include "hooking/hooking.hpp"
+#include "util/notify.hpp"
+
+namespace big
+{
+	uint32_t hooks::net_bundle_get_data_size(__int64 _this)
+	{
+		uint32_t ret = g_hooking->get_original<hooks::net_bundle_get_data_size>()(_this);
+
+		auto return_address = static_cast<uint8_t*>(_ReturnAddress());
+
+		if (return_address[0] == 0x33 && ret > 128) [[unlikely]]
```

Comment:
While this is a hook and it does satisfy #2 of the rules, does this hook get called enough to warrant telling the compiler what to do? I can't imagine this is called too often.